tag_manager/authorized_api/src/tags/schema/tag_templates/piwik_attributes.json

```diff
@@ -107,9 +107,10 @@
                     "type": "boolean"
                 },
                 "spent_time": {
-                    "description": "Time spent on page threshold in seconds",
+                    "description": "Time spent on page threshold in seconds (is ignored and will be removed in next version)",
```

Comment:
Does providing it in next version will generate error? Usually we give users more time to comply with the changes, but I don't think we have any standard for that.

Reply:
It will show up in the tag, so nothing will break on our side. As I understood the changes, it will be ignored by tracker. Am I correct? Next version here means API v2, which is not even planned yet.

Reply:
Currently we communicate to users that API version is same as PPAS version, this may cause confusion. Is there a way to explain it better?
![2021-04-27T175238](https://user-images.githubusercontent.com/4488774/116272982-aa9d7e00-a781-11eb-8d8d-8ef48184fc3a.png)